src/main/java/uk/gov/cshr/report/repository/CourseCompletionReportRequestRepository.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.cshr.report.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.cshr.report.domain.CourseCompletionReportRequest;
+
+import java.util.List;
+
+public interface CourseCompletionReportRequestRepository extends JpaRepository<CourseCompletionReportRequest, Long> {
+
+    @Query("""
```

Comment:
Does this need to be hardcoded query? The JPA helper methods should be able to cover this I'd imagine